products/compute/api.yaml

```diff
@@ -2288,25 +2256,18 @@ objects:
           subnetwork must be specified.
 
           This field is not used for external load balancing.
-      - !ruby/object:Api::Type::ResourceRef
-        name: 'region'
-        resource: 'Region'
-        imports: 'name'
-        description: |
-          A reference to the region where the regional forwarding rule resides.
-          This field is not applicable to global forwarding rules.
-        output: true
       # TODO(nelsonjr): Create a new type, WeakResourceRef that could accept
```

Comment:
It's probably worth copypasta-ing another reference to multi resource refs over this TODO so we can string search them.